bubblewrap.c

```diff
@@ -209,6 +210,7 @@ usage (int ecode, FILE *out)
            "    --dev-bind SRC DEST          Bind mount the host path SRC on DEST, allowing device access\n"
            "    --ro-bind SRC DEST           Bind mount the host path SRC readonly on DEST\n"
            "    --remount-ro DEST            Remount DEST as readonly; does not recursively remount\n"
+           "    --exec-filename PATH         Path to the application to be executed inside the sandbox\n"
```

Comment:
Perhaps `Executed instead of COMMAND, with COMMAND as its argv[0]` as a hint that this is for people with unusual requirements around `argv[0]`?